docs/ReferenceChanges.md

```diff
@@ -0,0 +1,50 @@
+Package: Microsoft.Extensions.Configuration.Binder
+  TFM: .NETStandard2.0
+    Removed:
+      Microsoft.Extensions.Configuration
```

Comment:
In this case, it turns out only code from M.E.Configuration.Abstraction is used. So the package removed the M.E.Configuration dependency and added a M.E.Configuration.Abstractions dependency.